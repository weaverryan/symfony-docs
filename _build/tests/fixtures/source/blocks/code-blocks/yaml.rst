
.. code-block:: yaml
    # some code
