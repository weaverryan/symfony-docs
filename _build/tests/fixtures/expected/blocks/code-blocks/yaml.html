<!DOCTYPE html>
<html>
<head>
    <meta charset="utf-8" />
</head>
<body>
<div class="literal-block">
    <div class="highlight-yaml">
        <table class="highlighttable">
            <tr>
                <td class="linenos">
                    <div class="linenodiv">
                        <pre> 1</pre>
                    </div>
                </td>
                <td class="code">
                    <div class="highlight">
                        <pre class="hljs yaml"><span class="hljs-comment"># some code</span></pre>
                    </div>
                </td>
            </tr>
        </table>
    </div>
</div>
</body>
</html>
